manifests/zabbixapi.pp

```diff
@@ -0,0 +1,33 @@
+# @summary This will install the zabbixapi gem.
+#
+# @param zabbix_version This is the zabbix version.
+# @param puppetgem Provider for the zabbixapi gem package.
+class zabbix::zabbixapi (
+  $zabbix_version = $zabbix::params::zabbix_version,
+  $puppetgem = $zabbix::params::puppetgem,
+) inherits zabbix::params {
+  # Determine correct zabbixapi version.
+  case $zabbix_version {
+    /^[56]\.[024]/: {
+      $zabbixapi_version = '5.0.0-alpha1'
+    }
+    default: {
+      fail("Zabbix ${zabbix_version} is not supported!")
+    }
+  }
+
+  $compile_packages = $facts['os']['family'] ? {
+    'RedHat' => ['make', 'gcc-c++', 'rubygems', 'ruby'],
+    'Debian' => ['make', 'g++', 'ruby-dev', 'ruby', 'pkg-config',],
+    default  => [],
+  }
+  ensure_packages($compile_packages)
```

Comment:
that is deprecated, please use the newer version:
```suggestion
  stdlib::ensure_packages($compile_packages)
```

Reply:
can I use this now w/o bumping stdlib to >= 9?

Reply:
I don't think so